Formula/g/go-parquet-tools.rb

```diff
@@ -0,0 +1,26 @@
+class GoParquetTools < Formula
+  desc "Utility to deal with Parquet data"
+  homepage "https://github.com/hangxie/parquet-tools"
+  url "https://github.com/hangxie/parquet-tools/archive/refs/tags/v1.25.9.tar.gz"
+  sha256 "61df81b68b58cfccd6b819bc3edf50e6ecf58e294195167517d71076fb047fd3"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  resource("test-parquet") do
+    url "https://github.com/hangxie/parquet-tools/raw/v1.25.9/testdata/good.parquet"
+    sha256 "daf5090fbc5523cf06df8896cf298dd5e53c058457e34766407cb6bff7522ba5"
+  end
+
```

Comment:
this can be moved down into the test block

Reply:
addressed.